Quadriga/src/main/java/edu/asu/spring/quadriga/service/impl/UserManager.java

```diff
@@ -335,6 +337,15 @@ public boolean addNewUser(AccountRequest request) throws QuadrigaStorageExceptio
         
         return success;
     }
+    
+    @Override
+    public boolean checkUserRequestExists(String username) throws QuadrigaStorageException{
```

Comment:
how often do you use this? do you really need an extra method if it only checks for null?